examples/classification/bank_customer_intent.py

```diff
@@ -113,9 +113,7 @@
         "label",
     ],
     "index2class": index2class,
-    "text_fields": [
-        "ft.onto.base_ontology.Body"
-    ],
+    "text_fields": ["ft.onto.base_ontology.Body"],
```

Comment:
I would say, revert every file that's not relevant to this PR. don't touch them

Reply:
I tried to revert it but I think it's because `black` is updated so it won't pass CI if I push the original commit.

Reply:
the thing is if the master passes it should pass. you should keep black to be the same version from local to your side so that you don't need to change anything unnecessary.